app/models/notify_user/base_notification.rb

```diff
@@ -191,10 +32,9 @@ def delay_time(options)
     self.channels = {
     }
 
-    ## Aggregation
-
-    class_attribute :aggregate_per
-    self.aggregate_per = 1.minute
+    ## Restricting the params to be sent
+    class_attribute :sendable_attributes
+    self.sendable_attributes = []
 
     ## True will implement a grouping/aggregation algorithm so that even though 10 notifications are delivered eg. Push Notifications
     ## Only 1 notification will be displayed to the user within the notification.json payload
```

Comment:
True, but I think the whole feature needs a rethink anyway, in terms of the param grouping and what not, and how to avoid the race conditions we've run into elsewhere.